frontend/src/components/MembersPage/MembersPage.js

```diff
@@ -28,7 +28,7 @@ function MembersPage() {
   };
 
   useEffect(() => {
-    const BACKEND_URL = "https://upe.sainayunipati.com/students/";
+    const BACKEND_URL = "http://localhost:8080/students/";
```

Comment:
This is another change you'd omit from your commit. The "main" branch is meant to be the final product. However, this is rewiring we did to simplify work on the developer side — if we merged it, we would break the site's front end.

Reply:
Got it Is there a way to omit just one part of this commit?